ldms/src/sampler/zfs_common/Makefile.am

```diff
@@ -0,0 +1,14 @@
+noinst_LTLIBRARIES =
+dist_man1_MANS =
+
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
```

Comment:
Please delete all of the lines before AM_CPPFLAGS.

Reply:
Done